src/xdocs/checks/javadoc/javadocpackage.xml

```diff
@@ -51,22 +51,101 @@
 
       <subsection name="Examples" id="Examples">
         <p id="Example1-config">
-          To configure the check:
+          To configure the check with no package-info.java file in directory:
```

Comment:
put it in `  <p id="Example1-structure">`

Reply:
Done